Makefile

```diff
@@ -104,27 +120,39 @@ upgrade-image.amd64: upgrade
 	 sudo docker build -t "${IMAGE_ORG}/${UPGRADE_REPO_NAME_AMD64}:${IMAGE_TAG}" ${DBUILD_ARGS} .
 	@rm build/${UPGRADE}/${UPGRADE}
 
-.PHONY: upgrade-image.arm64
-upgrade-image.arm64: upgrade
-	@echo "-----------------------------------------------"
-	@echo "--> ${UPGRADE} image                           "
-	@echo "${IMAGE_ORG}/${UPGRADE_REPO_NAME_ARM64}:${IMAGE_TAG}"
-	@echo "-----------------------------------------------"
-	@cp bin/${UPGRADE}/${UPGRADE} build/${UPGRADE}
-	@cd build/${UPGRADE} && \
-	 sudo docker build -t "${IMAGE_ORG}/${UPGRADE_REPO_NAME_ARM64}:${IMAGE_TAG}" ${DBUILD_ARGS} .
-	@rm build/${UPGRADE}/${UPGRADE}
-
-
-
-
 # cleanup upgrade build
 .PHONY: cleanup-upgrade
 cleanup-upgrade: 
 	rm -rf ${GOPATH}/bin/${UPGRADE}
 
+# build migrate binary
+.PHONY: migrate
+migrate:
+	@echo "----------------------------"
+	@echo "--> ${MIGRATE}              "
+	@echo "----------------------------"
+	@# PNAME is the sub-folder in ./bin where binary will be placed. 
+	@# CTLNAME indicates the folder/pkg under cmd that needs to be built. 
+	@# The output binary will be: ./bin/${PNAME}/<os-arch>/${CTLNAME}
+	@# A copy of the binary will also be placed under: ./bin/${PNAME}/${CTLNAME}
+	@PNAME=${MIGRATE} CTLNAME=${MIGRATE} CGO_ENABLED=0 sh -c "'$(PWD)/build/build.sh'"
+
+# build migrate image
+.PHONY: migrate-image.amd64
```

Comment:
```suggestion
.PHONY: migrate-image
```

Reply:
Lets keep this as it is as the normal build is only tested for amd64.